app/assets/stylesheets/layout/helper-classes.scss

```diff
@@ -460,6 +460,7 @@
   color: darken($smoke, 15%);
   font-style: italic;
   display: block;
+  padding-bottom: 0.5rem;
```

Comment:
my only question is, is there a way for us to use our $base-padding variable and some math to end up with 0.5rem? I just know we like to use that variable whenever possible.

Reply:
Sure, I can take a look